sound/soc/intel/boards/sof_sdw.c

```diff
@@ -617,6 +617,15 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		},
 		.driver_data = (void *)(SOC_SDW_CODEC_SPKR),
 	},
+	/* Pantherlake devices*/
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Intel_ptlrvp"),
+		},
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
```

Comment:
no, all devices after TGL only use 3 HDMI ports, this will add an HDMI port for no reason.

Reply:
Sure, i will remove.